templates/base.html

```diff
@@ -65,7 +65,6 @@
                             <li class="home-logo"><a class="small-home-logo" href="{% url 'home' %}"><img class="home-logo" src="{% static "img/perdiem-logo.svg" %}"></a></li>
                             <li><a class="small-nav-icon" href="{% url 'artists' %}"><i class="fa fa-search" aria-hidden="true"></i></a></li>
                             <li><a class="small-nav-icon" href="{% url 'music' %}"><i class="fa fa-music" aria-hidden="true"></i></a></li>
-                            <li><a class="small-nav-icon" class="nav-icon" href="{% url 'leaderboard' %}"><i class="fa fa-bar-chart" aria-hidden="true"></i></a></li>
```

Comment:
No, I think it's really great to have. I just don't think it's ready to be public facing yet. Would like to get some of those formatting issues worked out first. It's would be nice to have as a hidden link just for reference.